packages/flutter_tools/lib/src/commands/clean.dart

```diff
@@ -43,20 +43,16 @@ class CleanCommand extends FlutterCommand {
 
     deleteFile(flutterProject.dartTool);
 
-    final Directory androidEphemeralDirectory = flutterProject.android.ephemeralDirectory;
```

Comment:
Remove the locals for readability (when does that ever happen!?)

Reply:
On phone, by mistake.